src/map/packets.h

```diff
@@ -2229,6 +2229,17 @@ packet(0x96e,-1,clif->ackmergeitems);
 // changed packet sizes
 #endif
 
+// 2013-03-06aRagexe
+#if PACKETVER >= 20130306
+// new packets
+	packet(0x09a6,12); // ZC_BANKING_CHECK
```

Comment:
style in packets.h was always own. probably better left it as is.
About case, some times it small, some times in big. For checking tools here no difference. For humans too. My tool using always low case.